packages/keyring-snap-bridge/src/options.ts

```diff
@@ -36,3 +36,47 @@ export function getDefaultInternalOptions(): Required<SnapKeyringInternalOptions
     setSelectedAccount: true,
   };
 }
+
+/**
+ * Makes any field potentially `undefined`.
+ */
+export type PotentiallyUndefined<Type> = {
+  [Key in keyof Type]: Type[Key] | undefined;
+};
+
+/**
+ * Combine multiple internal options together. The internal options
+ * in the list will be merged in the order they are provided. If some options
+ * are still not defined, it will fallback to the default ones.
+ *
+ * @param internalOptions - List of internal options to combine.
+ * @returns Computed internal options.
+ */
```

Comment:
```suggestion
/**
 * Combine multiple internal options together. The internal options
 * in the list will be merged in the order they are provided. The first
 * appearance of a key is used, and the following are ignored. If some options
 * are still not defined, it will fallback to the default ones.
 *
 * @param internalOptions - List of internal options to combine.
 * @returns Computed internal options.
 */
```